### DISCUSSION TITLE
New Release for  prometheus-nginxlog-exporter

### DISCUSSION THREAD

**Discussion by jvineet** [martin-helmich/prometheus-nginxlog-exporter#190] [General]
@martin-helmich We have been using `prometheus-nginxlog-exporter` for a few months now and have been quite happy with it. I believe the new features you have merged recently, particularly around wildcard file names and json parsing are of particular interest to us in order to streamline some of our work better. I noticed there hasn't been a new release since October 2020, and I was hoping that you may consider cutting a new release, that includes the work at least up to this commit (https://github.com/martin-helmich/prometheus-nginxlog-exporter/commit/97f76776cbba25b546ba2371d05d41836d6eac64), in the coming days. We would really love to use the latest docker image that is part of the standard release process for this project rather than building and managing one from src on our end.

**Comment by siggy** [comment]
+1 to this, also `v1.8.0` does not include `only_counter` support.

**Reply by martin-helmich** [reply]
Thanks for the reminder, folks (and apologies for the late reaction) :+1: I've released [v1.9.0](https://github.com/martin-helmich/prometheus-nginxlog-exporter/releases/tag/v1.9.0) just now. 🚀
